python/src/lib.rs

```diff
@@ -8,7 +8,8 @@ use glide_core::Telemetry;
 use glide_core::MAX_REQUEST_ARGS_LENGTH;
 use pyo3::exceptions::PyTypeError;
 use pyo3::prelude::*;
-use pyo3::types::{PyAny, PyBool, PyBytes, PyDict, PyFloat, PyList, PySet, PyString};
+use pyo3::types::PyString;
+use pyo3::types::{PyAny, PyBool, PyBytes, PyDict, PyFloat, PyList, PySet};
```

Comment:
Why? no complains there from rust linter for such change? Interesting

Reply:
why are we making this change?